### PR TITLE
Impl Display for StoreId

### PR DESCRIPTION
**Pull request by matthiasbeyer**
@TheNeikos 

Does this make any sense here?

### CODE REVIEW

File: libimagstore/src/storeid.rs [hunk 1]
```diff
@@ -24,6 +24,17 @@ impl Into<PathBuf> for StoreId {
 
 }
 
+impl Display for StoreId {
+
+    fn fmt(&self, fmt: &mut Formatter) -> RResult<(), FmtError> {
+        match self.0.to_str() {
+            Some(s) => write!(fmt, "{}", s),
+            None    => write!(fmt, "<non-UTF8-StoreId>"), // TODO: Sure here?
```
Comment (TheNeikos) [inline libimagstore/src/storeid.rs]: You can use a lossy to str which will replace it with error chars

Comment (matthiasbeyer) [inline libimagstore/src/storeid.rs]: Woha, nice. Will open new PR, or do you want to do it?

Comment (TheNeikos) [inline libimagstore/src/storeid.rs]: I prob will forget it, feel free.

Comment (matthiasbeyer) [inline libimagstore/src/storeid.rs]: See https://github.com/matthiasbeyer/imag/pull/503

**Pull request merged**